docs/index.md

```diff
@@ -106,6 +106,17 @@ Use the the `irida-uploader.sh` script included with the source code to upload.
 
 `./irida-uploader.sh /path/to/the/sequencing/run/`
 
+## Logging
+
+Logs are written to your system default logging directory
+
+#### Linux
+
+`/home/<user>/.cache/irida_uploader/log/`
```

Comment:
Something i'm noticing is that when I run the tests it logs into this same file.  Any way we can get the tests to dump somewhere else?

Reply:
Added a environment variable (that does not persist after tests are run), to write tests logs to a different directory.

Added a note to the readme on where tests log.